logcat/api/logcat.api

```diff
@@ -12,33 +13,34 @@ public final class logcat/AndroidLogcatLogger$Companion {
 	public static synthetic fun installOnDebuggableApp$default (Llogcat/AndroidLogcatLogger$Companion;Landroid/app/Application;Llogcat/LogPriority;ILjava/lang/Object;)V
 }
 
-public final class logcat/AndroidLogcatLoggerKt {
-}
-
 public final class logcat/LogPriority : java/lang/Enum {
 	public static final field ASSERT Llogcat/LogPriority;
 	public static final field DEBUG Llogcat/LogPriority;
 	public static final field ERROR Llogcat/LogPriority;
 	public static final field INFO Llogcat/LogPriority;
 	public static final field VERBOSE Llogcat/LogPriority;
 	public static final field WARN Llogcat/LogPriority;
+	public static fun getEntries ()Lkotlin/enums/EnumEntries;
 	public final fun getPriorityInt ()I
 	public static fun valueOf (Ljava/lang/String;)Llogcat/LogPriority;
 	public static fun values ()[Llogcat/LogPriority;
 }
 
-public final class logcat/LogcatKt {
```

Comment:
This would be a source incompatible change for Java, but it's easy find/replace to fix it